pkg/menu/menu.go

```diff
@@ -166,7 +166,7 @@ func DisplayResult(message []string, uiEvents <-chan ui.Event) (string, error) {
 }
 
 // parsingMenuOption parses the user's operation in the menu page, such as page up, page down, selection. etc
-func parsingMenuOption(labels []string, menu *widgets.List, input, warning *widgets.Paragraph, uiEvents <-chan ui.Event) (int, error) {
+func parsingMenuOption(labels []string, menu *widgets.List, input, warning *widgets.Paragraph, customWarning []string, uiEvents <-chan ui.Event) (int, error) {
```

Comment:
you can do this in a simpler way.
func parsingMenuOption(labels []string, menu *widgets.List, input, warning *widgets.Paragraph, uiEvents <-chan ui.Event), customWarning ...string)

then it is a variadic. then you only need to change calls that want to use the warning:

Reply:
Got it! Thank you!